src/main-process/atom-protocol-handler.js

```diff
@@ -36,15 +36,23 @@ module.exports = class AtomProtocolHandler {
       let filePath;
       if (relativePath.indexOf('assets/') === 0) {
         const assetsPath = path.join(process.env.ATOM_HOME, relativePath);
-        const stat = fs.statSyncNoException(assetsPath);
-        if (stat && stat.isFile()) filePath = assetsPath;
+        try {
+          const stat = fs.statSync(assetsPath);
+          if (stat && stat.isFile()) filePath = assetsPath;
+        } catch () {
+          return false;
+        }
       }
 
       if (!filePath) {
         for (let loadPath of this.loadPaths) {
           filePath = path.join(loadPath, relativePath);
-          const stat = fs.statSyncNoException(filePath);
-          if (stat && stat.isFile()) break;
+          try {
+            const stat = fs.statSync(filePath);
+            if (stat && stat.isFile()) break;
+          } catch () {
```

Comment:
I think this is a syntax error, according to CI and from trying it in node's REPL. (Just running `node` and then typing it in to run it as a JS snippet).

From some quick testing, the `catch` needs an error bound to a var, or else you can omit the parentheses.

**Either of the below suggestions should be valid `catch` syntax:**

```suggestion
          } catch (e) {
```

^ This is valid, even if you don't so anything with `e`. This is the old way that has worked since forever.

```suggestion
          } catch {
```

^ With no `catch` error binding. This syntax is allowed since ~2018 in most browsers: https://caniuse.com/mdn-javascript_statements_try_catch_optional_catch_binding

And either is supported in our version of Electron based on Chrome 89. (https://www.electronjs.org/docs/latest/tutorial/electron-timelines)

Reply:
And incidentally this PR has several package tests in CI hitting this syntax error and then stalling for 6 hours, eating up our parallel runner allocation

D: